scenes/game/local_game.gd

```diff
@@ -1667,11 +1675,23 @@ func advance_to_next_turn():
 	var player_ending_turn = _get_player(active_turn_player)
 	var other_player = _get_player(get_other_player(active_turn_player))
 
-	# Do any end of turn effects.
+	# Do any end of turn character effects.
 	var effects = player_ending_turn.get_character_effects_at_timing("end_of_turn")
 	for effect in effects:
 		events += do_effect_if_condition_met(player_ending_turn, -1, effect, null)
 
+	# Do any end of turn boost effects.
+	# Iterate in reverse as items can be removed.
+	for i in range(len(player_ending_turn.continuous_boosts) - 1, -1, -1):
+		var card = player_ending_turn.continuous_boosts[i]
+		for effect in card.definition['boost']['effects']:
+			if effect['timing'] == "end_of_turn":
+				if card.id in player_ending_turn.end_of_turn_boost_delay_card_ids:
+					# This effect is delayed a turn, so remove it from the list and skip it for now.
```

Comment:
I wonder if it'd work nicely to add a field in "end_of_turn" effects to indicate how many turns they go off in, just to avoid needing to have a separate effect to add it to the delay list? Could have a dictionary somewhere to indicate how many turns the boost has skipped to tell when it should go off. I doubt there's anything with a longer waiting time than "end of your next turn" though so might be excessive

Reply:
This list could be a dictionary, I just didn't think there was a need for it.
I kind of wanted to track the turns remaining in the effect itself, but that just felt kind of nasty.
I think I'll just stick with this.